packages/core/src/generatePolicy.js

```diff
@@ -203,17 +214,25 @@ function createModuleInspector(opts) {
   /**
    * @param {AST} ast
    * @param {import('./moduleRecord').LavamoatModuleRecord} moduleRecord
-   * @param {boolean} includeDebugInfo
+   * @param {Partial<ModuleInspectorOptions>} options
    * @returns
    */
-  function inspectForEnvironment(ast, moduleRecord, includeDebugInfo) {
+  function inspectForEnvironment(
+    ast,
+    moduleRecord,
+    { includeDebugInfo = false, allowDynamicRequires = false } = {}
+  ) {
     const { packageName } = moduleRecord
     // @ts-expect-error `ParseResult` / `AST` mismatch
     const compatWarnings = inspectSesCompat(ast)
 
     const { primordialMutations, strictModeViolations, dynamicRequires } =
       // @ts-expect-error `SesCompat` / `InspectSesCompatResult` mismatch
       /** @type {import('./schema').SesCompat} */ (compatWarnings)
+
+    if (allowDynamicRequires && dynamicRequires.length) {
```

Comment:
```suggestion
    if (dynamicRequires?.length) {
```

Reply:
That seems to change the intention.  Why do this?

Reply:
Misread, sorry